src/TranslationUtils.gd

```diff
@@ -3,7 +3,8 @@ class_name TranslationUtils extends RefCounted
 static func get_shortcut_description(action_name: String) -> String:
 	match action_name:
 		"export": return TranslationServer.translate("Export")
-		"import": return TranslationServer.translate("Import")
+		"import": return TranslationServer.translate("Import SVG")
+		"import_reference_image": return TranslationServer.translate("Import Reference Image")
```

Comment:
I'd suggest using "Load reference image" without modifying the "Import" string.

Reply:
Done!